lesson_1.1/src/AddTodoForm.js

```diff
@@ -1,18 +1,27 @@
-import React from "react";
-import TodoList from "./TodoList";
-function AddTodoForm(props) {
+import React,{useState} from 'react'
+function AddTodoForm({onAddTodo}) {
+  const[todoTitle, setTodoTitle]= useState("");
+
+  const handleTitleChange= (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+    
+
+  };
+
+
```

Comment:
you don't need all of this blank space. one line between the handleTitleChange function and handleAddTodo is just fine